about.html

```diff
@@ -0,0 +1,109 @@
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>   
+            <div class="logo"><a href="index.html"><img src="images/Market (1).png" alt=""></a></div>
+            <ul class="nav">
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Program</a>
+                </li> 
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Join</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Sponsor</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">New</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link campaign">CC Campaign</a>
+                </li>
+            </ul>
+        </div>
+         
+       
+      </nav>
+      <section class="header">
+       <div class="header-1">
+        <h1 class="header-title"> Digital Marketing Capstone: A User-Friendly <br> Website to Manage and Monitor Smart Strategies</h1>
+    </div>
+        <p class="para-2">The primary aim of this project is to develop a user-friendly website that allows businesses and marketers to manage and monitor their digital marketing strategies effectively and efficiently. This platform enables users to have centralized control over their marketing endeavors and leverage the power of social networks to expand brand awareness and increase conversions.</p>
+       
+       <div class="para-3-cont">
+       <p>Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+       <span><a href=""> ccsummit2015@cckorea.org</a></span>
+       </div>
+       </section>
+
+        <section class="cont-4">
+       <h2>Creative Commons <br> Global Summit 2015 Logo<h2>
+        <div class="separator"></div>
+        <p class="para-3">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+        <p class="para-8">The logo of Digital Marketing 2011 was decided through the logo competition from 8, June to 7, July.
+            After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+            After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+        </div>
+        <div class="logo-2"><img src="images/Market (1).png" alt=""></div>
+        
+      </section>
+
+      <section class="program-2">
+        <div class="title-3">See the past CC Global Summits</div>
+        <div class="separator"></div>
+        <p> Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw </p>
+
+        <div class="img-col">
+            <img src="images/summit-1.png" alt="">
+            <img src="images/summit-1 (1).png" alt="">
+        </div>
```

Comment:
- **[OPTIONAL]** Great job on incorporating the feedback from the previous reviewers! You've really made some excellent improvements. 😊 I have a small optional suggestion. Could you please take a look at your mobile version? I noticed that in your about page, the images appear larger than the width of the page, causing them to break horizontally. If you can fix that, it would further enhance your project and give it an even more professional look. Keep up the good work!

   Video for a better understanding:
   
   [scrnli_8_5_2023_12-44-38 PM.webm](https://github.com/jerymey/Capstone/assets/107952724/103c82c7-9a59-4e9d-afe3-5cd75238a7dc)